modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -815,7 +815,7 @@ export const console: ConsoleNS = {
                                     },
                                     subHeading: "Select which user attributes you want to share with the application."
                                 },
-                                attributeComponentHint: "Use <1>OpenID Connect Scopes</1> to add/remove user attribute to a scope. " +
+                                attributeComponentHint: "Use <1>OpenID Connect Scopes</1> to manage user attribute in a scope. " +
```

Comment:
Don't we need to update this in other language files?

Reply:
Since this is a language specific decision I don't think we need to change.